README.md

```diff
@@ -107,6 +107,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 *Libraries for administrative interfaces.*
 
 * [ajenti](https://github.com/ajenti/ajenti) - The admin panel your servers deserve.
+* [django-admin-interface](https://github.com/fabiocaccamo/django-admin-interface) - Customizable responsive admin interface, based on a modern flat theme, it lets you customize the admin title, logo and colors by the admin itself. Popup windows replaced by modals.
```

Comment:
Hi @fabiocaccamo,
It looks like you accidentally included b3b45ce2c688feaa96cd6bb87d21cc2565640b95 into this PR.

Reply:
Ops, I'm sorry...